python/xorbits/pandas/tests/test_accessors.py

```diff
@@ -111,3 +112,38 @@ def test_obj_docstring(setup, dummy_str_series, dummy_dt_series):
         "This docstring was copied from "
         "pandas.core.indexes.accessors.CombinedDatetimelikeProperties."
     )
+
+    def f():
```

Comment:
No need to define a new function. You can just assert if the properties you added have the expected docstring.

Like,
```
assert dummy_df.loc.__doc__ is not None and dummy_df.loc.__doc__.endswith("This docstring was copied from ...")
```